src/index.js

```diff
@@ -32,35 +32,37 @@ const getLocalDate = date => {
 
 export const formatServerDate = dateObj => {
   if (typeof dateObj === 'string') throw new Error('Input is a string, expected a Date Object')
-  return isValid(dateObj) ? format(dateObj, FORMAT_SERVER_DATE) : ''
+  return isValid(dateObj) ? format(getUTCDate(dateObj), FORMAT_SERVER_DATE) : undefined
 }
 
 export const formatServerDateTime = dateObj => {
   if (typeof dateObj === 'string') throw new Error('Input is a string, expected a Date Object')
-  return isValid(dateObj) ? format(dateObj, FORMAT_SERVER_DATETIME) : ''
+  return isValid(dateObj) ? format(getUTCDate(dateObj), FORMAT_SERVER_DATETIME) : undefined
 }
 
 export const formatDisplayDateTime = dateObj => {
   if (typeof dateObj === 'string') throw new Error('Input is a string, expected a Date Object')
-  return isValid(dateObj) ? format(dateObj, FORMAT_DISPLAY_DATETIME) : ''
+  return isValid(dateObj) ? format(getUTCDate(dateObj), FORMAT_DISPLAY_DATETIME) : ''
```

Comment:
This seems like it would show the wrong time, since we are converting it to UTC.